### PR TITLE
Feat: Added Carousel

### PR DESCRIPTION
**Pull request by spraveenofficial**
Added Scrollable Carousel on Home Page.

As of Now, Categories Data is Static. Will move to API call very soon.

### CODE REVIEW

File: frontend/src/Components/Carousel/style.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
+.carousel-main-container {
+  background: var(--white-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 200px;
```
Comment (AmishMishra11) [inline frontend/src/Components/Carousel/style.css]: use rems instead of px

File: frontend/src/Components/Carousel/style.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}
+.carousel-wrapper {
+  width: 100%;
+  max-width: 900px;
```
Comment (AmishMishra11) [inline frontend/src/Components/Carousel/style.css]: same as above

File: frontend/src/Components/Carousel/style.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  margin: auto;
+  box-sizing: border-box;
+}
+
+.carousel-items {
+  width: 100%;
+  height: 100%;
+}
+.carousel-items img {
+  width: 130px;
```
Comment (AmishMishra11) [inline frontend/src/Components/Carousel/style.css]: same as above

**Review by AmishMishra11 — COMMENTED** [review]
Clean code , made some suggessions

**Review by tanishk15sharma — APPROVED** [review]
The code is looking appropriate to me . suggestion you can import all the images in one file.

**Comment by spraveenofficial** [comment]
> tanishk15sharma

I have used a public folder to store my images, there we need to give only the image path. We don't need to import images.

**Pull request merged**